widget/ui/src/components/Typography/Typography.styles.tsx

```diff
@@ -33,6 +33,11 @@ export const TypographyContainer = styled('span', {
         whiteSpace: 'nowrap',
       },
     },
+    selectNone: {
```

Comment:
Please don't every css property a prop. and also make it boolean will cause to add `selectAll`, `selectText` and other possible values in future.

Reply:
You can cover you use case in playground by doing something like this.

```
const NotSelectableTypograph = styled(Typograph, { ... })
```

Reply:
Added "NotSelectableTypograph" as a component to UI repository.